pillar/params.sls

```diff
@@ -115,9 +115,8 @@ etcd:
 
 # the flannel backend ('udp', 'vxlan', 'host-gw', etc)
 flannel:
-  backend:        'udp'
-  etcd_key:       '/flannel/network'
-  iface:          'eth0'
+  image:          'sles12/flannel:1.0.0'
+  backend:        'udp'   # TODO: UDP seems to be near end of life (https://github.com/coreos/flannel/pull/786)
```

Comment:
I would say to change this to vxlan right now but there may be upgrade implications.

Reply:
Maybe we should expose the ability to choose to users ASAP, and default to vxlan or something - i.e. start weaning new deployments off UDP mode!

Reply:
I am in favour of switching to vxlan now, but that will break connectivity when upgrading from non-CNI to CNI...

Reply:
Well - I meant, changing the default, rather than changing existing deployments. We can set that pillar to "udp" during the upgrade orch (maybe? Can we set pillars in the DB from here?), if it's not already set, which will keep UDP for existing deployments, but, change the default to vxlan and allow the user to choose during a deployment.